src/index.js

```diff
@@ -4,9 +4,8 @@ import ReactDOM from 'react-dom';
 import App from './App';
 
 ReactDOM.render(
-<React.StrictMode>
-  <App />
-</React.StrictMode>, 
-document.getElementById('root')
+  <React.StrictMode>
```

Comment:
[OPTIONAL] The app crashes when someone tries to use the module `%` operand with 0, as it is technically a division by 0. You can correct this issue in the same way that `operate.js` corrects the division by zero.

![image](https://user-images.githubusercontent.com/79114233/144269511-87e43c28-f3de-458a-98ee-4b300254d597.png)